### ISSUE TITLE
Braking Doesn't Work Properly

### ISSUE THREAD

**Issue by geoff-white** [johnsie/HoverNet#21]
The hard-break (by default SHIFT+DOWN) does quickly slow down the hovercrafts, but no longer brings them to a complete stop where they stay stopped anymrore.

I vaguely remember a previous developer mentioning that it didn't make sense to them that flipping the turbine would stop the craft and hold it in place, so this may have been a, "feature," that was added, but which I think most HoverRace veterans would agree breaks some of the functionality and nostalgia of the game.

If this was done purposely, it is something that could be considered for a non-Classic version, but Classic should try to stick with the original controls as much as possible.

**Comment by johnsie** [comment]
I totally agree and will attempt to rectify this if possible. The quirks with physics in the HoverRace universe are important

**Comment by johnsie** [comment]
Confirmed. It makes the craft change direction which I don't think is correct. I've put this into milestone 1 because I think it's quite important that its fixed..

**Comment by johnsie** [comment]
I found some code that handles braking put changing it back to the original didn't stop the craft from rotating and accelerating. I have a feeling that there is some other code somewhere that handles this. Will keep looking. I would like this to be fixed during phase 1 if possible
